<launch>

	<arg name="cam_name"   default="camera_right_eye"/>


	<!-- - - - - - - - - - H A R D W A R E - - - - - - - - - - - - - -->
	<!-- camera right eye -->
	<include file="$(find bender_sensors)/launch/camera/$(arg cam_name).launch">
		<arg name="display" value="false"/>
		<arg name="cam_number" value="0"/>
	</include>



	<!-- - - - - - - - - - - - N a v i g a t i o n - - - - - - - - - - -->
	<!--include file="$(find bender_nav)/launch/complete_nav.launch">
	  <arg name="simulation" value="true"/>
	  <arg name="rviz"       value="true"/>
	</include-->

	<!-- - - - - - - - - - - - - - S p e e c h - - - - - - - - - - - - -->
	<include file="$(find bender_speech)/launch/complete_speech.launch"/>

	
	<!-- - - - - - - - - - - - - - V i s i o n  - - - - - - - - - - - -->

	<!-- Blob Filter maco>40y 100-->
	<node name="blob_filter" pkg="bender_vision" type="blob_filter" ns="/bender/vision/" output="screen">
		<param name="cam_name" value="$(arg cam_name)"/>
		<param name="color_thresh" value="120"/>
		<param name="color_thresh_max" value="180"/>
		<param name="min_blob_size" value="50"/>
	</node>

	<node name="blob_filter_display" pkg="bender_vision" type="blob_filter_display" output="screen" required="false" ns="/bender/vision/">
		<param name="cam_name" value="$(arg cam_name)"/>
	</node>

	<node  name="emergency_detector" pkg="bender_vision" type="emergency_detector" ns="/bender/vision/">
		<param name="cam_name" value="$(arg cam_name)"/>
	</node>

</launch>
